docs/docs/components/README.md

```diff
@@ -0,0 +1,108 @@
+- Deploy via Pipedream's UI, CLI or API
+
+**Example**
+
+The Search Mentions source for Twitter is a pre-built component in Pipedream's registry that can be deployed in seconds and emits new Tweets that match a search query. After a user configures and deploys it, the deployed component polls the Twitter API for new results. When new results are found, each new result is emitted as an event that may be inspected and used to trigger a workflow.
+
+### Actions
+
+Actions are components that may be used as steps in a workflow. Unlike sources, actions cannot run independently (outside of a workflow).
+
+**Capabilities**
+
+- Accept user input via `props`
+- May `return` JSON serializable data
+
+**Example**
+
+The Add Single Row action for Google Sheets is a pre-built component in Pipedream's registry that can be added to a workflow and configured in seconds. Users can configure it in seconds and send workflow data to Google Sheets without having to learn the Google Sheets API.
+
+## Using Components
+
+Components may be instantiated or added to workflows via Pipedream's UI. 
+
+- Sources may be instantiated and consumed via [UI](https://pipedream.com/sources/new), CLI or API
+- Actions may be added to [workflows](https://pipedream.com/new)
+
+## Developing Components
+
+Develop components locally using your preferred code editor (and maintain your code in your own Github repo) and deploy or publish using Pipedream's [CLI](/cli/reference/#pd-deploy). 
+
+- Sources may be deployed directly from local code or published to your account and instantiated via Pipedream's UI
+- Actions may only be published — published actions may be added to workflows via Pipedream's UI
+
+Published components are only available to your own account by default. If published to a team account, the component (source or action) may be discovered and selected by any member of the team. 
+
+### Prerequisites
+
+- A free [Pipedream](https://pipedream.com) account 
+- A free [Github](https://github.com) account
+- Basic proficiency with Node.js or Javascript
+- Pipedream [CLI](/cli/reference/)
+
+Finally, the target app must be integrated with Pipedream. You can explore all apps supported by Pipedream in the [marketplace](https://pipedream.com/explore). If your app is not listed, please [create a Github issue](https://github.com/PipedreamHQ/pipedream/issues/new?assignees=&labels=app%2C+enhancement&template=app---service-integration.md&title=%5BAPP%5D) to request it and [reach out](https://pipedream.com/community/c/dev/11) to our team to let us know that you're blocked on source or action development.
+
+### Quickstart Guides
+
+- [Sources](quickstart/nodejs/sources/) 
+- [Actions](quickstart/nodejs/actions/)
+
+### Component API Reference
+
+After getting familiar with source/action development using the quickstart guides, check out [the Component API Reference](COMPONENT-API.md) and [examples on Github](https://github.com/pipedreamhq/pipedream/components) to learn more.
```

Comment:
The component API reference is now `/components/api`, correct?

Reply:
correct! done